src/Plugin_Command.php

```diff
@@ -672,7 +672,7 @@ protected function get_item_list() {
 
 				// Compare version and update information in plugin list.
 				if ( null !== $plugin_update_info && version_compare( $details['Version'], $plugin_update_info->new_version, '>' ) ) {
-					$items[ $file ]['update'] = 'version higher than expected';
+					$items[ $file ]['update'] = self::INVALID_VERSION_MESSAGE;
```

Comment:
As the class is not `final`, it's safer to use `static` here, otherwise extension can produce unexpected results...
```suggestion
					$items[ $file ]['update'] = static::INVALID_VERSION_MESSAGE;
```